admin/view/template/extension/payment/cardinity_order_ajax.twig

```diff
@@ -85,34 +89,34 @@ $('.button-command').on('click', function() {
 			dataType: 'json',
 			beforeSend: function() {
 				$('#button-settle').button('loading');
-
-				$('.alert').hide();
-
-				$('.alert').removeClass('alert-success alert-danger');
+				$('#refund_alert').hide();
+				$('#refund_alert').removeClass('alert-success alert-danger');
 			},
 			complete: function() {
 				$('#button-settle').button('reset');
 			},
 			success: function(json) {
+                console.log("Testing");
```

Comment:
Please remove logging statements here and below, as it will be shown in the customers browser